packages/app-project/src/screens/ClassifyPage/components/YourStats/YourStatsContainer.js

```diff
@@ -19,7 +20,7 @@ function useStoreContext(stores) {
 }
 
 function YourStatsContainer({ stores }) {
-  const { counts, projectName } = useStoreContext(stores)
+  const { counts, projectName } = useStores(stores)
```

Comment:
```suggestion
  const { counts, projectName } = useStores(stores.store)
```

This might need changing everywhere that you've changed the arguments to the hook.

Reply:
TBH, the tests could just pass in the mock store as a `store` prop, rather than passing `stores`.

Reply:
The components where I changed the arguments to a hook were based on the example of ProjectHeaderContainer's `useStores()`, and those components work in the app locally.

I hadn't looked at why tests are failing in this PR yet. If `useStores()` works in the browser, but the component's test is failing, is the test setup correct?

Reply:
The `stores` prop here is being passed in by the tests. Stories might use it too.

Reply:
When reading through components with `useStores()` I've always found the naming of `stores` and `store` confusing because it's not immediately clear this code is to handle a testing environment. I went back and renamed some to `mockStore` if the component needed refactoring due to linting in this PR, but I'll plan to follow up with another PR that unifies the `useStores()` syntax for all components in app-project.

Reply:
Agreed. I’ve always found `stores.store` confusing too. I assume the store is set up like that to allow for more than one store, but then why is it `stores.store` and not `stores.projectStore`? The second option leaves room for additional stores like `stores.userStore`.

Reply:
We can also drop those mock `store` props when components are converted to tests that support `useContext`, like you’ve done with the About page components.